### PR TITLE
allow activating invite for the other identities

### PR DESCRIPTION
**Pull request by RealMahmoud**

### CODE REVIEW

**Comment by sidenaio** [comment]
Thank you for your contribution. Please run gofmt to format the code.

**Comment by ridenaio** [comment]
Hi! I think we have several things to improve here. 
- We can rename `Payload` field to `PubKey` in API args because the only public key can be passed;
- We can remove 'To' field from params and calculate it from pubKey, I mean:

```
var pubKey []byte
if args.PubKey != nil {
	pubKey = *args.PubKey 
} else {
	pubKey = api.baseApi.secStore.GetPubKey()
}
to, err := crypto.PubKeyBytesToAddress(pubKey)
```

**Comment by RealMahmoud** [comment]
Ok The last build passed 😃

File: api/dna_api.go [hunk 1]
```diff
@@ -144,13 +144,15 @@ func (api *DnaApi) ActivateInvite(ctx context.Context, args ActivateInviteArgs)
...
-		to = &coinbase
+
+	var pubKey []byte
+	if args.PubKey != nil {
+		pubKey = *args.PubKey
+	} else {
+		pubKey = api.baseApi.secStore.GetPubKey()
 	}
-	hash, err := api.baseApi.sendTx(ctx, from, to, types.ActivationTx, decimal.Zero, decimal.Zero, decimal.Zero, args.Nonce, args.Epoch, payload, key)
+	to, err := crypto.PubKeyBytesToAddress(pubKey)
```
Comment (ridenaio) [inline api/dna_api.go:154]: Please, check `err` here in order to prevent request panic

Comment (RealMahmoud) [reply api/dna_api.go:154]: Done. 👍🏻

**Review by ridenaio — APPROVED** [review]

**Review by sidenaio — APPROVED** [review]

**Pull request merged**